heka/_service.sls

```diff
@@ -2,7 +2,7 @@
 
 {%- set server = salt['pillar.get']('heka:'+service_name) %}
 
-{%- if server.enabled %}
+{%- if server is defined %}
```

Comment:
Enabled is important for removing this service

Defined/enabled both has own role